testing/unit_tests/test_main.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+from ivadomed.main import check_multiple_raters
+
+@pytest.mark.parametrize(
+    'is_train, loader_params', [
+       (False, {"target_suffix":
+           [["_seg-axon-manual1", "_seg-axon-manual2"],
+            ["_seg-myelin-manual1", "_seg-myelin-manual2"]]
+           }) 
+])
+def test_check_multiple_raters(is_train, loader_params):
```

Comment:
Thank you for adding this!
You are right, I forgot that `--test` is not working with multiple raters right now, sorry about that.
In any case, adding this test is a great idea so we don't forget about this feature in the future, thanks again!